tutorials/clustering.md

```diff
@@ -22,15 +25,92 @@ need a common superuser role on each postgres instance. This tutorial
-Some or most of these may not be necessary depending on how you
-initially set up your instances:
-- Make sure you've enabled `max_prepared_transactions` on all the instances, this parameter is located in `postgresql.conf`. We recommend a setting of at least 150.
-- Make sure the data nodes are properly listening for network traffic. `postgresql.conf` should have a `listen_addresses` set to include the access node. If this parameter is missing or just `localhost`, change it to `access.example.com`, or the corresponding IP. You can also set it to `"*"` to not restrict where connections can originate from.
-- The superuser role, `postgres` in this example, must have an entry in the `pg_hba.conf` file for the data nodes. This should look something like: ```host    all    postgres    access.example.com    trust``` You can use any CIDR block covering the access node's IP in place of `access.example.com`. For using authentication mechanism's other than `trust`, please see the [authentication guide][data-node-authentication].
+### Update your Postgres configuration
+
+First of all, if you haven't yet run the `timescaledb-tune` utility, it's
+highly recommended that you do so before starting. Do this for all of your
+nodes.
+```
+sudo timescaledb-tune
+```
+
+Also, while you're visiting each of your nodes, make sure you've enabled
+prepared transactions on them, as these are essential for our distributed
+transactions. By default these are disabled, so add the following line to
+your `postgresql.conf`.
+```
+max_prepared_transactions = 150
+```
+
+### Set password encryption
+
+The next thing we'll be doing is setting up the network communication between
+our nodes and adding authentication. For more details and option for this step,
+please visit the [data node authentication][data-node-authentication] page.
+
+In this example we'll be adding `scram-sha-256` password authentication to your
+nodes and adding a password entry for the postgres role as well as a new user
+role we'll be creating later on. The first step of this is to enable password
+encryption on all the nodes by adding the following to `postgresql.conf`:
+
+```
+password_encryption = 'scram-sha-256'
+```
+
+>:WARNING: This setting is not automatically applied to any existing passwords.
+Any existing roles will have to update to new passwords to take advantage of
+the encryption.
+
+After making this change, restart the postgres service so the changes made so
+far can take effect.
+
+### Set up the access node
+
+The next step is to enable `access.example.com` to send the passwords for the
+various users of the system. Create a new file `passfile` in the Postgres data
+directory, and add the following lines:
+```
+*:*:*:postgres:postgres
+*:*:*:testuser:testpass
+```
+
+This will result in the access node always using password `postgres` for role
+`postgres` and `testpass` for role `testuser` whenever connecting to a data
+node. Make sure that this file is readable by the postgres user.
```

Comment:
Note that there is a difference here between the `postgres` user on the system (OS) and the `postgres` user in the DB. In fact, the former might not exist at all on some OSes (i.e., postmaster runs under another user name). I guess what you want to say is that the whatever user the postmaster runs under needs to be able to read this file.